WalletWasabi/Wallets/PasswordFinder.cs

```diff
@@ -0,0 +1,68 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+
+namespace WalletWasabi.Wallets
+{
+	public static class PasswordFinder
```

Comment:
Don't make it static.

Reply:
See my comment here: https://github.com/zkSNACKs/WalletWasabi/pull/4960#discussion_r547331506